### PR TITLE
Add wazo-purge-db plugin

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: integration_tests/suite/test_database.py [hunk 1]
```diff
@@ -146,3 +148,42 @@ def on_subscription(service, _):
...
+                3,
+                started_at,
+                started_at + datetime.timedelta(minutes=1),
+                {},
+                {},
+            )
+
+        for i in range(5):
+            add_log(i)
+            add_log(i)
```
Comment (fblackburn1) [inline integration_tests/suite/test_database.py]: refactor to be more explicit, ex:
```python
for days in range(5):
    add_log(days_ago=days)
    add_log(days_ago=days)
```

File: integration_tests/suite/test_database.py [hunk 2]
```diff
@@ -146,3 +148,42 @@ def on_subscription(service, _):
...
+
+        logs = service.get_logs(subscription_uuid)
+        assert_that(len(logs), 10)
+
+        with service.rw_session() as session:
+            SubscriptionLogsPurger().purge(2, session)
```
Comment (fblackburn1) [inline integration_tests/suite/test_database.py]: be more explicit on magic number. ex:
```suggestion
            days_to_purge = 2
            SubscriptionLogsPurger().purge(days_to_purge, session)
```

**Comment by fblackburn1** [comment]
The documentation from wazo-doc should be updated or refactored to mention this table and/or the plugin system with the default config of 30 days

**Comment by sileht** [comment]
https://github.com/wazo-pbx/wazo-doc/pull/21

**Comment by sileht** [comment]
> Add a number to the config drop-in file to be easily overwritable
Done

> Should we add a debian dependency between webhookd and purge-db?
In my ideal world, the debian package will be splitted in two parts:
* wazo-wbhookd (contains console-script, /etc, systemd

**Comment by fblackburn1** [comment]
> 
> I think wazo-purge-db should depends on python-wazo-webhookd-common (since it doesn't exists yet, we could put wazo-webhookd, is that ok ?
>
Since the dependency is not really high, I don't have a real opinion, is up to you :grin:

**Pull request merged**